main/java/TestForRealItemClass.java

```diff
@@ -0,0 +1,18 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class TestForRealItemClass extends BeforeAfter {
+
+    @Test
+    void scenarioSetRealItemWeightAndPrice() {
```

Comment:
RealItem and VirtualItem classes were designed as simple classes without any business logic. There is no needs to write tests for these classes at all, the only method for which it can be done - toString(). Do not write tests for simple operations (get/set). Please remove test and add for toString() (not required)

Reply:
In Task 10 there was a task to develop 1 test for RealItem class and 1 test for VirtualItem class, and I selected the simplest methods I could check. 
I will re-do.

Reply:
Correction is applied.

Reply:
It was a trap) Now I believe you will remember that for long time